README.md

```diff
@@ -147,7 +147,7 @@ The local development environment for the static assets can be started by execut
 
 This will run the React application at [http://localhost:8080](http://localhost:8080) -- please see `Serverless Framework` below for enabling the 'server' side functionality.
 
-### Serverless Framework
+### Serverless Framework Locally
```

Comment:
There is a markdown linter warning about having two sections with the same names. There is a another section in the README called `Serverless Framework` this one is specifically referring to the dev env

Reply:
I think this might read better as "Running Serverless Framework locally"